### PR TITLE
Fixing CI errors

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: docker/dockerfile-build-centos [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    pkgconfig \
+    numactl-libs 
+
+RUN echo '#!/bin/bash' | tee /etc/profile.d/devtoolset7.sh && echo \
+    'source scl_source enable devtoolset-7' >>/etc/profile.d/devtoolset7.sh
+
+# docker pipeline runs containers with particular uid
+# create a jenkins user with this specific uid so it can use sudo priviledges
+# Grant any member of sudo group password-less sudo privileges
+RUN useradd --create-home -u ${user_uid} -o -G video --shell /bin/bash jenkins && \
```
Comment (saadrahim) [inline docker/dockerfile-build-centos:38]: isn't this what you want?

Comment (saadrahim) [reply docker/dockerfile-build-centos:38]: I am trying to install a pacakge

Comment (saadrahim) [reply docker/dockerfile-build-centos:38]: @eidenyoshida

File: docker/dockerfile-build-ubuntu-rock [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    libnuma1 \
+    libomp-dev \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# docker pipeline runs containers with particular uid
+# create a jenkins user with this specific uid so it can use sudo priviledges
+# Grant any member of sudo group password-less sudo privileges
+RUN useradd --create-home -u ${user_uid} -o -G sudo --shell /bin/bash jenkins && \
```
Comment (saadrahim) [inline docker/dockerfile-build-ubuntu-rock:39]: @eidenyoshida maybe it is missing here.

**Comment by saadrahim** [comment]
@eidenyoshida can you take a look at the cron, I want this to run daily

File: .jenkins/precheckin.groovy [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        if (urlJobName == jobName)
+            stage(jobName) {
+                runCI(nodeDetails, jobName)
+            }
+    }
+
+    // For url job names that are not listed by the jobNameList i.e. compute-rocm-dkms-no-npi-1901
+    if(!jobNameList.keySet().contains(urlJobName))
+    {
+        properties(auxiliary.addCommonProperties([pipelineTriggers([cron('0 1 * * *')])]))
```
Comment (eidenyoshida) [inline .jenkins/precheckin.groovy:76]: 0 1 * * 0 is 1AM on Sundays I believe
0 1 * * * is 1AM every day

**Pull request merged**